docs/quickstarts/twitter.rst

```diff
@@ -5,7 +5,7 @@ Set up the application
 ----------------------
 Go to Twitter Application Manager at https://apps.twitter.com and create a
 new app. The application's "Callback URL" must be
-``http://localhost:5000/login/twitter/authorized``.
+``http://127.0.0.1:5000/login/twitter/authorized``. The domain ``localhost`` will not work with Twitter!
```

Comment:
Can you point me to some documentation for Twitter's OAuth implementation that supports this change?

Reply:
@singingwolfboy I found it [here](https://twittercommunity.com/t/why-cant-i-use-localhost-as-my-oauth-callback/708) - I also assumed localhost would work until I tried it - in the app callback settings, it even accepts localhost without a validation error but it fails when I tried it use it.